examples/vertical-fl/README.md

```diff
@@ -106,412 +22,67 @@ seconds to run as the model is very small.
-with the return type of the `fit` function, we need to return a list of
-`np.array`s (hence the conversion), the number of samples, which won't be used
-on the server side, so we just return 1, and then an empty dict.
-
-For the `evaluate` function, we perform our model's backward pass using the
-gradients sent by the server and then update our local model's parameters based
-on those new gradients. Note that the `loss` and `num_examples` we return in our
-evaluate function are bogus, as they won't be used on the server side.
+## Run the project
 
-The `client_fn` we will use in our `start_simulation` function to generate our 3
-clients will be very basic:
+You can run your Flower project in both _simulation_ and _deployment_ mode without making changes to the code. If you are starting with Flower, we recommend you using the _simulation_ mode as it requires fewer components to be launched manually. By default, `flwr run` will make use of the Simulation Engine.
 
-```python3
-partitions, label = get_partitions_and_label()
+### Run with the Simulation Engine
 
-def client_fn(cid):
-    return FlowerClient(cid, partitions[int(cid)]).to_client()
+```bash
+flwr run .
 ```
 
-We pass a `client_id` and its corresponding partition to each client.
+You can also override some of the settings for your `ClientApp` and `ServerApp` defined in `pyproject.toml`. For example:
 
-### Evaluation
-
-Please note that we do not perform distributed evaluation. This is because only
-the server holds some labels to compare the results to. This is why the only
-evaluation we perform is on the server side.
-
-In this example, we use the `FlowerClient` `evaluate` function for
-backpropagation instead of using it for evaluation. We do this because we know
-that the `evaluate` function of the clients will be called after the fit
-function. This allows us to aggregate our models in `aggregate_fit` and then
-send them back to the clients using this `evaluate` function and perform the
-backpropagation. This is not done for evaluation, hence why we return `None` in
-the `aggregate_evaluate` function of the strategy.
-
-### Starting the simulation
-
-Putting everything together, to start our simulation we use the following
-function:
-
-```python
-hist = fl.simulation.start_simulation(
-    client_fn=client_fn,
-    num_clients=3,
-    config=fl.server.ServerConfig(num_rounds=1000),
-    strategy=Strategy(label),
-)
+```bash
+flwr run . --run-config "num-server-rounds=5 learning-rate=0.05"
 ```
 
-As mentioned before, we train for 1000 rounds but it should still last only
-a few seconds.
-
-Note that we store the results of the simulation into `hist`, this will allow us
-to use the `plot.py` file to plot the accuracy as a function of the number of
-rounds.
-
-## Results
+> \[!TIP\]
+> For a more detailed walk-through check our [vertical FL tutorial](https://flower.ai/docs/framework/tutorial-vertical-fl.html)
```

Comment:
@charlesbvll are you planning to include a tutorial? currently this link isn't working. Should we remove and bring it back when the tutorial is live?

Reply:
@jafermarq It should work once it is deployed, the tutorial is added in this PR